### ISSUE TITLE
Clojure bindings

### ISSUE THREAD

**Issue by solicode** [breach/thrust#269]
I created Clojure bindings for Thrust: [clj-thrust](https://github.com/solicode/clj-thrust).

I also created a small [sample project](https://github.com/solicode/clj-thrust-example-notepad) with setup instructions for anybody who wants to get and up running quickly.

I'm having some people currently test it, but once that's done, would it be okay to for me to add Clojure to the README? I don't know if you'd like to eventually create a separate page for the details for the language bindings as it's getting quite big, but either way is perfectly fine with me.

Also, I'd like to add a thank you for the API. I found it quite easy to get this all working. (^^)

**Comment by miketheprogrammer** [comment]
Yes it would be ok. Just submit a PR, and @spolu will probably merge when he gets a chance.

**Comment by miketheprogrammer** [comment]
And your welcome, I am sure you thank you makes spolu and the other contrib's happy.

**Comment by solicode** [comment]
Great, thanks! I submitted the PR.
